library/src/main/java/org/xmtp/android/library/Group.kt

```diff
@@ -215,6 +220,38 @@ class Group(val client: Client, private val libXMTPGroup: FfiGroup) {
         return libXMTPGroup.updateGroupName(name)
     }
 
+    fun isAdmin(inboxId: String): Boolean {
+        return libXMTPGroup.isAdmin(inboxId)
+    }
+
+    fun isSuperAdmin(inboxId: String): Boolean {
+        return libXMTPGroup.isSuperAdmin(inboxId)
+    }
```

Comment:
Can we have another function on member that tells the permission level for the member? https://github.com/xmtp/xmtp-android/blob/main/library/src/main/java/org/xmtp/android/library/libxmtp/Member.kt

Reply:
like this idea, add here dab330a326c523b378191d2ac7656563eb1f7d16